src/provider/CoinbaseWalletProvider.ts

```diff
@@ -1124,14 +1126,16 @@ export class CoinbaseWalletProvider
       })
     }
 
+    const chainId = parseInt(this.chainId, 16)
```

Comment:
seems we could use `getChainId` function directly without parsing it to an int @pedro-rosario

Reply:
`this.chainId` is a formatted version of `getChainId`
![image](https://user-images.githubusercontent.com/966662/157145073-70373950-3039-40ff-88e0-49ad987ee6c8.png)